pages/tracking.js

```diff
@@ -0,0 +1,9 @@
+import TrackingSection from "../components/TrackingSection";
+
+export default function TrackingPage() {
+  return (
+    <>
+      <TrackingSection />
+    </>
```

Comment:
If this is the only place where you've used this component (which is simply a `p` tag with one word inside), don't use a component here. Besides, why is there only one word on this entire page?

Reply:
As mentioned before, just a placeholder. Sorry for that.